.changeset/tasty-seahorses-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': patch
+---
+
+Extend the DocsTable component to accept custom user-defined options while still providing default values in case of empty.
```

Comment:
It would be nice to add a code snippet as an example here...

Reply:
thanks for the review,code snippet added as an example.